testsuite/cluster-test/src/suite.rs

```diff
@@ -40,6 +40,9 @@ impl ExperimentSuite {
         experiments.push(Box::new(
             PerformanceBenchmarkParams::new_nodes_down(10).build(cluster),
         ));
+        experiments.push(Box::new(
+            PerformanceBenchmarkParams::new_fixed_tps(0,50).build(cluster),
```

Comment:
So the TPS we would care about in this experiment is something lower, like maybe 5 TPS